CondFormats/PhysicsToolsObjects/test/SiStripDeDxMipBuilder.h

```diff
@@ -22,7 +22,7 @@ class SiStripDeDxMipBuilder : public edm::EDAnalyzer {
   virtual void analyze(const edm::Event&, const edm::EventSetup&);
 
 private:
-  edm::FileInPath fp_;
+  //edm::FileInPath fp_;
```

Comment:
NOTE: this was left intentionally commented out as the code that uses fp_ was already commented out.